libc-test/semver/netbsd.txt

```diff
@@ -1275,8 +1275,10 @@ jrand48
 kevent
 key_t
 killpg
-kinfo_vmentry
```

Comment:
Why was this removed?

Reply:
It wasn't removed, I just sorted this hunk - it's under kinfo_proc2.